pkg/nodejs/pnpm/parse.go

```diff
@@ -64,13 +69,29 @@ func (p *Parser) parse(lockFile *LockFile) ([]types.Library, []types.Dependency)
 	var libs []types.Library
 	var deps []types.Dependency
 
+	var lockVer float64
+	switch v := lockFile.LockfileVersion.(type) {
+	// lock
+	case float64:
+		lockVer = v
+	case string:
+		var err error
+		if lockVer, err = strconv.ParseFloat(v, 64); err != nil {
+			log.Logger.Debugf("unable to convert lock file version: %s", err)
```

Comment:
Log message should be capitalized.

```suggestion
			log.Logger.Debugf("Unable to convert lock file version: %s", err)
```

Reply:
Made in https://github.com/aquasecurity/go-dep-parser/pull/220/commits/2babc94911560f26355064c29a8340c4765ad992